ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -347,6 +439,63 @@
       // Process features
       this._queryFeatures = this._createFeatureIndex(data.query.features);
 
+      this.renderResults();
+    },
+
+    createPageLink : function(page, selected) {
+        var pageSpan = new Element("a", {'class':'pagenumber', 'href':'#'}).update(page);
+        if (selected) {
+           pageSpan.addClassName("selected");
+        }
+        var _this = this;
+        pageSpan.observe("click", function(ev){
+            ev.stop();
+            _this.page = ev.element().innerHTML;
+            _this.launchSearchExisting();
+        });
+        return pageSpan;
+    },
+
+    generatePaginationPageLinks : function() {
```

Comment:
the question from @allasm that he originally asked in my first implementation: if `this.totalPages` is 1000+, will all page links be displayed at once?